commit 16944e3ea5e5962349f6099e3be7b977cebd3616
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2019-04-12T21:16:33-07:00 (6 months ago)
    
    joyent/triton#252 Typo on README.md where physical install indicates COAL download

diff --git a/README.md b/README.md
index eaf272b..2afbd11 100644
--- a/README.md
+++ b/README.md
@@ -128,7 +128,7 @@ the [Joyent Manufacturing Database](https://docs.joyent.com/private-cloud/hardwa
 To install Triton, first download the latest release image:
 
 ```bash
-curl -C - -O https://us-east.manta.joyent.com/Joyent_Dev/public/SmartDataCenter/coal-latest.tgz
+curl -C - -O https://us-east.manta.joyent.com/Joyent_Dev/public/SmartDataCenter/usb-latest.tgz
 ```
 
 Once you have downloaded the latest release image, you will need to
